### ISSUE TITLE
[Question] How does sysbench expose the C program interface to Lua scripts in version 1.1.0

### ISSUE THREAD

**Issue by 8a2a8** [akopytov/sysbench#500]
As stated in the title，In sysbench version 1.1.0
I can't find a way for the sysbench program to expose the C interface to Lua scripts

**Comment by 8a2a8** [comment]
Taking the function sb_lua_set_test_args() as an example, this function is located at src/sb_lua.c, line 95, sysbench 1.1.0
This function is used to register custom parameters for testing Lua scripts
Its call stack is as follows: main() -> sb_load_lua() -> read_cmdline_options() -> ffi.C,sb_lua_set_test_args()(in lua script) -> Omitting the Lua virtual machine call section ->  sb_lua_set_test_args()
I have tried to verify the following guesses:
1. c register functions with Luajit virtual machine: when debugging sysbench using gdb, I break at lua_pushcclosure(), it only called by luaL_newstate() and luaL_openlibs(), and I couldn't find the register function in the file
2. lua load liarary, when debugging sysbench using gdb, i break at dlopen(), it never called, and I couldn't find the load function in the lua file
3. load at compile time: I have tried to imitate the way sysbench compiles and runs, like`gcc test_lib.o main.o -ldl`, c program call lua script, lua script call c function, but it has not been effective, and the result showed that it couldn't find the corresponding c function
